src/components/Tool/Appeal/Appeals.tsx

```diff
@@ -8,19 +8,21 @@ import {
 } from '@material-ui/core';
 import { useTranslation } from 'react-i18next';
 import { useGetAppealsQuery } from '../../../../pages/accountLists/[accountListId]/tools/GetAppeals.generated';
-import { useAccountListId } from '../../../../src/hooks/useAccountListId';
 import Appeal from '../../../../src/components/Tool/Appeal/Appeal';
 import NoAppeals from '../../../../src/components/Tool/Appeal/NoAppeals';
 
 const LoadingIndicator = styled(CircularProgress)(({ theme }) => ({
   margin: theme.spacing(0, 1, 0, 0),
 }));
 
-const Appeals = (): ReactElement => {
-  const accountListId = useAccountListId();
+interface Props {
+  accountListId: string;
+}
+
+const Appeals: React.FC<Props> = ({ accountListId }: Props) => {
   const { t } = useTranslation();
   const { data, loading } = useGetAppealsQuery({
-    variables: { accountListId: accountListId || '' },
+    variables: { accountListId: accountListId },
```

Comment:
I think for all these cases you can just do: 
```suggestion
    variables: { accountListId },
```